cmd/http/listener.go

```diff
@@ -120,11 +120,15 @@ func getResourceHost(bufConn *BufConn, maxHeaderBytes int) (resource string, met
 
 		if method == "" && resource == "" {
 			if i := strings.IndexByte(tokens[0], ' '); i == -1 {
-				return "", "", "", fmt.Errorf("malformed HTTP request %s, from %s", tokens[0], bufConn.LocalAddr())
+				return "", "", "", fmt.Errorf("malformed HTTP request, from '%s'", bufConn.LocalAddr())
```

Comment:
This message needs to be beautified.

Reply:
can you provide an example?

Reply:
Use error message as fmt.Errorf("malformed HTTP request from '%s'", bufConn.LocalAddr())

Reply:
fixed